.github/workflows/piplock-renewal.yaml

```diff
@@ -0,0 +1,65 @@
+---
+# This GitHub action is meant to update the pipfile.locks
+name: Pipfile.locks Renewal Action
+
+on:  # yamllint disable-line rule:truthy
+  workflow_dispatch:  # for manual trigger workflow from GH Web UI
+    inputs:
+      branch:
+        description: 'Specify branch'
+        required: false
+        default: 'main'
+      python_version:
+        description: 'Select Python version to update Pipfile.lock'
+        required: false
+        default: '3.11'
+        type: choice
+        options:
+          - '3.11'
+          - '3.9'
+          - '3.8'
+      update_optional_dirs:
+        description: 'Include optional directories in update'
+        required: false
+        default: 'false'
+        type: choice
+        options:
+          - 'true'
+          - 'false'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      # Checkout the specified branch from the specified organization
+      - name: Checkout code from the specified branch
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.inputs.branch }}
+          token: ${{ secrets.GH_ACCESS_TOKEN }}
```

Comment:
why we have the naming different to what we use in [kubeflow](https://github.com/opendatahub-io/kubeflow/pull/489/files#diff-9c97925bf1481f29538c73d5dd78b0639e2629a148e381aec814d25669f0f3b5R87)? Is there some reason for this? Note that I like this naming more than the one in the kubeflow TBH.

Reply:
This was the old original name that this GHA used. Thus, I keep the same name, but now I issued the token from the `ide-developer` account. The old one I have a feeling that issued by personal account.

Reply:
Okay, but maybe it would make sense to unify this among the repositories in the future to avoid confusion as I guess these are both same things with just different naming, right? :slightly_smiling_face: